tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -484,6 +489,16 @@ def test_pdb_check_simultaneous_node_drains(
         time.sleep(30)
         try:
             drain_nodes([node_B])
+            # Check Mon failovers in 10mins, 15mins and 20 mins
+            timeout = [600, 300, 300]
```

Comment:
The intervals mentioned in the comment are 10mins, 15mins and 20 mins. So, the timeout values here should be
timeout = [600, 900, 1200]

Reply:
Yes 20 mins going to be whole timeoujt, I am checking that in intervals of 10, 15 and 20 mins and not totally 10+15+20 mins. Hence the intervals are 600+300+300 totally 20 mins

Reply:
Ok, the added comment is confusing, can you please elaborate and add a clear mention about it.